packages/ruby/lib/readme/http_request.rb

```diff
@@ -134,5 +128,17 @@ def host_header
         'Host' => @request.host
       }.compact
     end
+
+    def read_body(io)
```

Comment:
@AndriiAndreiev what's the context behind this method?

Reply:
I didn't find any exact solutions for this, but this method seems to fix it

Reply:
@AndriiAndreiev can you add a test case for this? Looks good to me, but I'd love some additional confidence in our code base.